### PR TITLE
Better docs for --ignore-bad-template

### PR DESCRIPTION
**Pull request by stampflit**
*Issue #, if available:*

*Description of changes:*

Adjust the cli info string and the Readme to contain the actual behaviour and additionally the recommended usage of the `--ignore-bad-template` flag

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Optional parameters:
 | -f, --format    | format | quiet, parseable, json, junit, pretty, sarif | Output format |
 | -l, --list-rules | | | List all the rules |
 | -r, --regions | regions | [REGIONS [REGIONS ...]], ALL_REGIONS  | Test the template against many regions.  [Supported regions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) |
-| -b, --ignore-bad-template | ignore_bad_template | | Ignores bad template errors |
+| -b, --ignore-bad-template | ignore_bad_template | | Suppress the `Template file not found` error. This is useful for suppressing errors caused by wildcards that expand to no files |
```
Comment (kddejong) [inline README.md:181]: I believe this will suppress all errors in decoding.  So if the yaml/json can't be decoded this will also suppress it.

Comment (stampflit) [reply README.md:181]: Are you sure about that?

minimal.yaml
```yaml
---
AWSTemplateFormatVersion: "2010-09-09"
Resources::
  # Helps tests map resource types
  IamPipeline:
    DeletionPolicy: Delete
    UpdateReplacePolicy: Delete
    Type: "AWS::CloudFormation::Stack"
    Properties:
      TemplateURL: !Sub "https://s3.${AWS::Region}.amazonaws.com/bucket-dne-${AWS::Region}/${AWS::AccountId}/pipeline.yaml"
      Parameters:
        DeploymentName: iam-pipeline
        Deploy: "auto"
```
Note the second colon after `Resources`.

```bash
$ cfn-lint minimal.yaml
E1001 Missing top level template section Resources
minimal.yaml:1:1

E1001 Top level template section Resources: is not valid
minimal.yaml:3:1


$ echo $?
2

$ cfn-lint -b minimal.yaml
E1001 Missing top level template section Resources
minimal.yaml:1:1

E1001 Top level template section Resources: is not valid
minimal.yaml:3:1


$ echo $?
2

```

Comment (stampflit) [reply README.md:181]: Whereas it does work for suppressing file not found errors:

```bash
$ cfn-lint nonexistent.yaml
2022-11-23 08:06:26,955 - cfnlint.decode - ERROR - Template file not found: nonexistent.yaml
2022-11-23 08:06:26,956 - cfnlint.decode - ERROR - Template file not found: nonexistent.yaml
E0000 Template file not found: nonexistent.yaml
nonexistent.yaml:1:1


$ echo $?
2

$ cfn-lint -b nonexistent.yaml
2022-11-23 08:06:36,318 - cfnlint.decode - ERROR - Template file not found: nonexistent.yaml
2022-11-23 08:06:36,318 - cfnlint.decode - ERROR - Template file not found: nonexistent.yaml

$ echo $?
0
```

Comment (stampflit) [reply README.md:181]: You are right.  Should I add an additional option suppressing the error only when no template is found?

**Pull request closed without merging**